tests/client/bulk.rs

```diff
@@ -14,8 +14,8 @@ fn bulk_ordered_insert_only() {
         .map(|i| {
             WriteModel::InsertOne {
                 document: doc! {
-        "_id" => (i),
-        "x" => (i * 11)
+        "_id": i,
```

Comment:
God dammit, rustfmt...

I think these ones are bad enough that they should be manually fixed

Reply:
fixed this one manually.